### DISCUSSION TITLE
Single Sign On - 502 Gateway Error

### DISCUSSION THREAD

**Discussion by Torquemonster260** [inventree/InvenTree#4013] [General]
Hi all,
When I add the appropriate backend in config.yaml and then restart the server. I get a 502 gateway error.  Anyone run into this before?  I cant access the admin page to add the required client configurations in the SocialApp app.

Thanks

**Comment by matmair** [comment]
@Torquemonster260 do you have some more info for us? What deployment method are you using? Did it work before you added the backend? What backend did you add?
Also, 502 probably means that your backend server is not running.

**Reply by Torquemonster260** [reply]
@matmair I am using an ubuntu server with docker deployment.  Everything worked before I added the SSO backend and I was able to login and use Inventree. I tried adding github backend, microsoft and others separately to see if any worked but all caused the 502 error. 

Do I have to do something else to confirm the changes? Should I have an SSL certificate?
